packages/tokenbank_v1/contracts/modules/security/SecurityModule.sol

```diff
@@ -38,6 +38,26 @@ contract SecurityModule is MetaTxModule
         securityStore = _securityStore;
     }
 
+    // overriding
+    modifier onlyFromWalletOwner(address wallet) {
+        require(Wallet(wallet).owner() == msg.sender, "NOT_FROM_WALLET_OWNER");
+        _;
+        securityStore.touchLastActive(wallet);
```

Comment:
Any reason why only in SecurityModules? Or is it just because `SecurityModule` is basically the base class of all modules anyway?

Reply:
SecurityModule has access to SecurityStorage, so it knows if the wallet is locked, who are the guardians.  SecurityModule also supports meta-tx. So basically most modules should inherit from SecurityModule.